src/clientsettingsdlg.cpp

```diff
@@ -166,6 +167,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     cbxLOutChan->setAccessibleName ( tr ( "Left output channel selection combo box" ) );
     cbxROutChan->setWhatsThis ( strSndCrdChanMapp );
     cbxROutChan->setAccessibleName ( tr ( "Right output channel selection combo box" ) );
+#endif
 
     // enable OPUS64
     chbEnableOPUS64->setWhatsThis (
```

Comment:
Random point of note.

The UI has "(preferred)" against 128 samples.  The `strSndCrdBufDelay` text says `64 samples: The preferred setting`
![image](https://user-images.githubusercontent.com/1549463/148592884-979d73cb-cfe6-49af-b4d1-484abbc8868e.png)
![image](https://user-images.githubusercontent.com/1549463/148592991-cdcb44c1-90fc-43d4-bc60-14e95a851316.png)
Clever...

Reply:
Well spotted! The website does not include the "preferred" text for this text. Agree this should be a separate issue.